Readme.MD

```diff
@@ -183,7 +185,7 @@ Usabilla.canDisplayCampaigns = false
 Setting it to `true` will allow the SDK to display any campaign when it triggers.
 Setting it to `false` will prevent all campaigns from being displayed.
 
-**Even though `canDisplayCampaigns` is set to `false`, the SDK continues to receive the events sent and if a campaign happens to trigger, it won't be displayed nor delayed: That event instance will be lost.**
+**Even when `canDisplayCampaigns` is set to `false`, the SDK continues to receive all events sent to it and, if a campaign happens to trigger, it won't be displayed nor delayed: That event instance will be lost.**
```

Comment:
...the SDK continues to receive all events. If a campaign happens to trigger, it won't be displayed nor delayed: that event instance will be lost.